.github/workflows/spark.yaml

```diff
@@ -52,38 +52,44 @@ jobs:
             spark-hadoop: "spark-3.3.4-bin-hadoop3"
 
     steps:
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@v4
       with:
           submodules: true
 
     - name: Code Format Check
+      working-directory: spark
       run: |
         export JAVA_HOME=${JAVA_HOME_11_X64}
-        pushd spark
         mvn --no-transfer-progress spotless:check
-        popd
 
     - name: Build GraphAr Spark
+      working-directory: spark
       run: |
         export JAVA_HOME=${JAVA_HOME_11_X64}
-        pushd spark
         echo "Build ${{ matrix.mvn-profile }}"
-        mvn --no-transfer-progress clean package -DskipTests -Dspotless.check.skip=true -P ${{ matrix.mvn-profile }}
-        popd
+        mvn clean package -DskipTests -Dspotless.check.skip=true -P ${{ matrix.mvn-profile }}
+    
+    - name: Build Spark Docs
+      working-directory: spark
+      run: |
+        export JAVA_HOME=${JAVA_HOME_11_X64}
+        echo "Build ${{ matrix.mvn-profile }}"
+        # FIXME: the install is not necessary, but it is a workaround for the issue
+        mvn install -DskipTests -Dspotless.check.skip=true
```

Comment:
I would like to add --no-transfer-progress

Reply:
fixed